src/test/scala/com/exasol/cloudetl/orc/OrcConverterTest.scala

```diff
@@ -16,7 +16,7 @@ import org.scalatest.BeforeAndAfterEach
 import org.scalatest.funsuite.AnyFunSuite
 
 @SuppressWarnings(Array("org.wartremover.warts.AsInstanceOf"))
-class OrcDeserializerTest extends AnyFunSuite with BeforeAndAfterEach with DummyRecordsTest {
+class OrcConverterTest extends AnyFunSuite with BeforeAndAfterEach with FileManager {
```

Comment:
Should it be named  `OrcConverterPrimitiveTypesTest`?

Reply:
Do you test all primitive types somewhere? I can't find it

Reply:
Thanks for pointing this out, going to add one for primitive types.

I also realized there are couple test files used for unit test, they should be converted to unit tests.